docs/hacking.md

```diff
@@ -27,7 +27,7 @@ It uses the following Javascript technologies and principles:
   - direct use in browser `<script>` tags through bundling
   - bundling through webpack
 
-[Epoch's Swagger file]: https://github.com/aeternity/epoch/blob/master/config/swagger.yaml
+[Node's Swagger file]: https://github.com/aeternity/epoch/blob/master/config/swagger.yaml
```

Comment:
The repository url has changed to https://github.com/aeternity/aeternity/blob/master/config/swagger.yaml

Reply:
The url of the `swagger.yaml` still refers to the "old" repo